README.rst

```diff
@@ -99,6 +97,41 @@ check out our `docs <https://simplepath.readthedocs.org>`_.
         'to': 'people',
     }
 
+Here is another example if your data is stored in an object.
```

Comment:
I'd definitely put it in the docs, but probably not the README.

The README is really aimed at showing potentially interested developers how this lib can be useful to them, and it should be concise and direct so they don't lose interest before understanding what *simplepath* is. Adding documentation and examples on how to use utility methods would probably not be useful at that stage.